pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rich_text_view
 description: A simple yet powerful rich text view that supports mention, hashtag, email, url and see more.
-version: 1.2.1-dev-1
+version: 1.2.2-dev-1
```

Comment:
Could you remove this change please? I will update the necessary files for deploy